client/webserver/site/src/css/application.scss

```diff
@@ -10,6 +10,8 @@ $light_body_bg: white;
 $dark_body_bg: #091a28;
 $light_border_color: #ddd;
 $dark_border_color: #383f4b;
+$light_input_border: #999;
+$dark_input_border: #555;
```

Comment:
OK, this seems like the established place for these vars, but it would be nice (in the future) if the variables declared here _before_ the bootstrap imports were just the ones intended to customize bootstrap variables (https://getbootstrap.com/docs/5.0/customize/sass/#variable-defaults)  Any others that are dex specific would be better defined after the bootstrap imports, just before the `./` imports below.
I can no longer tell which of these 18 variables fall into that category.

Reply:
Merged but I think we should look at the placement of these vars to avoid ambiguity about whether they are local to the application, or doing sass customization of vars used internal to bootstrap.